### PR TITLE
Added error prop

### LINKED ISSUE #127: Add an error prop to highlight error state
- Add an `error` prop for fixed menu editor (ref: https://github.com/bigbinary/neeto-desk-web/issues/11327#issuecomment-1001529408). 

- When the prop is set, highlight an error state for the editor and display the passed message.

- Snippet for reference: https://github.com/bigbinary/neeto-editor/blob/master/lib/index.js#L23-L31

- Address these concerns after merge: https://github.com/bigbinary/neeto-desk-web/issues/11327#issuecomment-1000470579, [neeto-kb-web#653 (review)](https://github.com/bigbinary/neeto-kb-web/pull/653#discussion_r764921768).

enhancement _t discussion _t

### PR DESCRIPTION
**Pull request by mohit-s96**
closes #127 

- User of the Editor can now pass an `error` prop that will update the error state of the fixed editor.

### CODE REVIEW

**Comment by mohit-s96** [comment]
@AbhayVAshokan _a. Please review.

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@mohit-s96 _a please make the following changes and assign @karthiknmenon for the UI review.

https://vimeo.com/712110227/9aad5d97e8

**Comment by mohit-s96** [comment]
@karthiknmenon _a. Please review the UI changes. Let me know if the error message needs to be repositioned.

neeto-editor-error-state

File: lib/components/Common/ErrorWrapper.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
```
Comment (karthiknmenon) [inline lib/components/Common/ErrorWrapper.jsx:2]:
```suggestion
import classnames from "classnames";
```

File: lib/components/Common/ErrorWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      message = error;
+    } else if (is(Array, error)) {
+      message = error[0];
+    } else if (is(Object, error)) {
+      message = error.message;
+    }
+
+    if (isNilOrEmpty(message)) return null;
+
+    return <p className="ne-input__error">{message}</p>;
```
Comment (karthiknmenon) [inline lib/components/Common/ErrorWrapper.jsx:26]: I would prefer just returning the error message from here and not the `p` element. 
Use the message in the `return` of this component.

Comment (karthiknmenon) [reply lib/components/Common/ErrorWrapper.jsx:26]: cc: @AbhayVAshokan

**Review by karthiknmenon — CHANGES_REQUESTED** [review]
@mohit-s96 _a Please check. 
Also, please match the error styles with that of neetoUI Input Component.

**Comment by mohit-s96** [comment]
@karthiknmenon _a. Please review. I compared the error message with neetoUI input and they both match.

**Comment by AbhayVAshokan** [comment]
@karthiknmenon please provide an update on this PR. It has been open for quite some while now.

**Review by karthiknmenon — APPROVED** [review]

**Pull request merged**